### ISSUE TITLE
Getting warnings about Application.get_env/2 in test support files

### ISSUE THREAD

**Issue by warmwaffles** [elixir-sqlite/ecto_sqlite3#90]
When using Elixir 1.14.0 with OTP 25, we get a new set of warnings when running `mix test`

Here are the warnings.

```
warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:29: Ecto.Integration.Post

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:85: Ecto.Integration.Comment

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:109: Ecto.Integration.Permalink

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:133: Ecto.Integration.PostUser

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:151: Ecto.Integration.User

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:182: Ecto.Integration.Custom

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:200: Ecto.Integration.Barebone

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:216: Ecto.Integration.Tag

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:269: Ecto.Integration.Order

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:286: Ecto.Integration.CompositePk

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:307: Ecto.Integration.CorruptedPk

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:322: Ecto.Integration.PostUserCompositePk

warning: Application.get_env/2 is discouraged in the module body, use Application.compile_env/3 instead
  deps/ecto/integration_test/support/schemas.exs:339: Ecto.Integration.Usec
```

**Comment by joeljuca** [comment]
I just realized that it was reported on Sep 8, 22. After a bit of investigation, I reproduced this error in `v0.8.2`. It's a warning in Ecto itself, not in `ecto_sqlite3` – and it was probably fixed by 62dda12 or 6f1ca67.

In any case, I can't reproduce it in `v0.10.2`. Should we close the issue?

FYI, the command I used to find relevant changes in `mix.lock` was:

```
git log --after=2022-09-01 --grep=ecto mix.lock 
```
